txpipe/plotting/correlations.py

```diff
@@ -133,13 +133,13 @@ def axis_setup(a, i, j, ny, ymin, ymax, name):
     if name.startswith(r'C_\ell'):
         a.set_xlabel(r"$C_\ell$")
     else:
-        a.set_xlabel(r"$\theta / $ arcmin")
+        a.set_xlabel(r"$\theta$ [arcmin]")
 
     a.tick_params(axis='both', which='major', length=10, direction='in')
     a.tick_params(axis='both', which='minor', length=5, direction='in')
 
     # Fix
-    a.text(0.1, 0.1, f"Bin {i}-{j}", transform=a.transAxes)
+    a.text(0.1, 0.1, f"{j+1} - {i+1}", transform=a.transAxes)
```

Comment:
We had a vote on the 3x2pt channel and in LSST bins officially start from zero!
https://lsstc.slack.com/archives/C9JSJV3FZ/p1521045279000638

Reply:
Oh sorry about this I had no idea! Will correct it and push it again.